vllm/model_executor/layers/quantization/compressed_tensors/compressed_tensors.py

```diff
@@ -232,7 +232,8 @@ def _get_scheme_from_parts(
                 return CompressedTensorsWNA16(
                     num_bits=weight_quant.num_bits,
                     strategy=weight_quant.strategy,
-                    group_size=weight_quant.group_size)
+                    group_size=weight_quant.group_size,
+                    actorder=weight_quant.actorder)
```

Comment:
could we just add the condition here?
```actorder=weight_quant.actorder == ActivationOrdering.GROUP```

Reply:
We could, but it feels more logical to me to keep argument processing within the `CompressedTensorsWNA16.__init__` function. This separates responsibilities and makes clear that the job of `_get_scheme_from_parts` is to decide which compression scheme applies, not to process the arguments once the scheme is decided.

Reply:
We'd also have to rename the `actorder` argument of `CompressedTensorsWNA16.__init__`, otherwise it would be a misnomer